extensions/amp-story/1.0/animation.js

```diff
@@ -47,7 +47,14 @@ const ANIMATE_IN_DELAY_ATTRIBUTE_NAME = 'animate-in-delay';
 const ANIMATE_IN_AFTER_ATTRIBUTE_NAME = 'animate-in-after';
 /** const {string} */
 const ANIMATABLE_ELEMENTS_SELECTOR = `[${ANIMATE_IN_ATTRIBUTE_NAME}]`;
-
+/** const {string} */
+const ZOOM_START_ATTRIBUTE_NAME = 'zoom-start';
+/** const {string} */
+const ZOOM_END_ATTRIBUTE_NAME = 'zoom-end';
```

Comment:
Just like for `translate-y`, I like the idea of having an API that looks like the CSS properties it's going to set. Should we do `scale-` instead of `zoom-`?

Reply:
Done.